src/logtab.cpp

```diff
@@ -905,8 +917,13 @@ void LogTab::UpdateStatusBar()
                 status += ", ";
             }
             status += highlightOpt.m_value;
+            if (status.size() > 100)
+            {
+                status += "...";
+                break;
+            }
         }
-        status += "}";
+        status += "}  ";
```

Comment:
I see (and now I can't unsee it hehe). I think we should handle the padding through the UI though, not by adding artificial spaces to the strings.